components/ProtectedImage.vue

```diff
@@ -19,5 +25,15 @@ watch(
 );
 </script>
 <template>
-  <img :src="source" :alt="alt || ''" loading="lazy" v-bind="$attrs" />
+  <img
+    :src="source"
+    :alt="alt || ''"
+    loading="lazy"
+    v-bind="$attrs"
```

Comment:
I know it was used before... but if this is for the `class` keyword, this can be removed.

Reply:
done